src/core/main.qml

```diff
@@ -121,6 +158,8 @@ Window {
                 rccBackgroundMusic = ApplicationInfo.getBackgroundMusicFromRcc()
                 for(var i = 0; i < rccBackgroundMusic.length; i++)
```

Comment:
can you shuffle the music here before adding them to the list (Core.shuffle(rccBackgroundMusic))?

Reply:
yep, sure! I'll do it tomorrow (my exams ends tomorrow :D ) Thanks ;)